py-polars/tests/unit/operations/namespaces/array/test_array.py

```diff
@@ -63,6 +63,36 @@ def test_arr_sum(
     assert s.arr.sum().to_list() == expected_sum
 
 
+def test_array_lengths() -> None:
+    # Basic test with fixed width arrays
+    s = pl.Series("a", [[1, 2], [3, 4]], dtype=pl.Array(pl.Int64, 2))
```

Comment:
We can reduce this to one test. As it should only be dependent on the datatype, nulls, inner types etc don't really matter.